### PR TITLE
Rails setup

### PR DESCRIPTION
**Pull request by Joberg95**
#### Description:
Scaffolded rails app

### Included:
- skip test
- skip bundle
- skip coffee
- db postgres

#### What I have learned working on this feature: 
I learned how to include multiple skips when generating a rails app.

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -0,0 +1,55 @@
+source 'https://rubygems.org'
+git_source(:github) { |repo| "https://github.com/#{repo}.git" }
+
+ruby '2.4.1'
+
+# Bundle edge Rails instead: gem 'rails', github: 'rails/rails'
```
Comment (magnus-thor) [inline Gemfile]: Remove commented out code.

**Review by magnus-thor — CHANGES_REQUESTED** [review]

File: Gemfile (hunk 1 above)
Comment (Joberg95) [reply Gemfile]: Removed comments from gemfile.

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1,34 @@
+# See https://help.github.com/articles/ignoring-files for more about ignoring files.
```
Comment (magnus-thor) [inline .gitignore]: Remove the comments from this file as well

**Review by magnus-thor — CHANGES_REQUESTED** [review]

**Review by perhegg — APPROVED** [review]
LGTM!

File: Gemfile [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+group :development do
+  gem 'web-console', '>= 3.3.0'
+  gem 'listen', '>= 3.0.5', '< 3.2'
+  gem 'spring'
+  gem 'spring-watcher-listen', '~> 2.0.0'
+end
+
+
+gem 'tzinfo-data', platforms: [:mingw, :mswin, :x64_mingw, :jruby]
```
Comment (magnus-thor) [inline Gemfile]: This can be removed if no one in the team is on a windows machine

**Review by magnus-thor — APPROVED** [review]
Good job

**Pull request merged**